workshop/workshop-resources.md

```diff
@@ -38,199 +22,35 @@ On this page, we've assembled some resources you may find helpful during these s
-Make sure your current directory is your home directory by entering the following into Terminal and hitting Enter:
-
-```sh
-# Navigate to your home directory in Terminal
-cd ~
-```
-
-Then you're ready to copy the file with the following command:
-
-```sh
-cp -avr shared-data/working-with-your-data/retrieve-SRAdb-metadata.Rmd
-```
-
-You can open the Rmd file as normal.
-
-End SRAdb section -->
-
-## Transcriptome indices for common organisms
-
-During the introduction to bulk RNA-seq module, we used human data and included a transcriptome index for human in `training-modules/RNA-seq/index/`.
-
-If you have non-human RNA-seq data you would like to quantify, or want to experiment with slightly different index parameters, we have prepared indices for select organisms relevant to the study of childhood cancer.
-Note that for most of these, you will need to perform a few extra steps to read in the quantification data with `tximeta` after performing quantification.
-Please see the notebook [`RNA-seq/00c-tximeta_other_species.Rmd`](https://github.com/AlexsLemonade/training-modules/blob/master/RNA-seq/00c-tximeta_other_species.Rmd) for details on how to set this up.
-
-If you have RNA-seq data for an organism that is not listed, please post in the training-specific Slack channel and let your instructors know.
-
-### _Homo sapiens_
-
-Ensembl GRCh38 (hg38) v95
-
-| File description | File use | File path |
-|------------------|----------|-----------|
-| Human Salmon index `-k 23` | Salmon index for use with `salmon quant`; appropriate for reads shorter than 75bp or for increased sensitivity with `--validateMappings` ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/hg38_cdna/salmon_index/short` |
-| Human Salmon index `-k 31` | Salmon index for use with `salmon quant`; appropriate for reads 75bp or longer ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/hg38_cdna/salmon_index/long` |
-
-
-
-### _Mus musculus_
-
-Ensembl GRCm38 (mm10) v95
-
-| File description | File use | File path |
-|------------------|----------|-----------|
-| Mouse Salmon index `-k 23` | Salmon index for use with `salmon quant`; appropriate for reads shorter than 75bp or for increased sensitivity with `--validateMappings` ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/mm10_cdna/salmon_index/short` |
-| Mouse Salmon index `-k 31` | Salmon index for use with `salmon quant`; appropriate for reads 75bp or longer ([docs](https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode)) | `~/shared-data/reference/refgenie/mm10_cdna/salmon_index/long` |
-
-### _Danio rerio_
+## Resources for getting started with R programming
```

Comment:
The "Resources for" is a bit redundant in these three headers. Maybe nest these something like:

```suggestion
## Topic-specific resources 

### Getting started with R programming
```

and similar for the below sections?

Reply:
can do!